README.md

```diff
@@ -35,6 +31,19 @@ It is a free monthly webinar about "everything Serverless", an initiative from [
    - Powerpoint presentation: [https://www.slideshare.net/SrushithR/getting-started-with-amazon-eventbridge](https://www.slideshare.net/SrushithR/getting-started-with-amazon-eventbridge)
    - Resource Person: [Srushith Repakula](https://twitter.com/SrushithR), AWS Serverless Hero and Head of Engineering, KonfHub
 
+### 27th Feburary, 2021
+
+#### Session 1: Convince your boss to go Serverless
+   - Powerpoint presentation: [https://www.slideshare.net/VadymKazulkin/convince-your-boss-to-go-serverless-at-aws-user-group-tirupathi-and-serverless-bangalore-meetup](https://www.slideshare.net/VadymKazulkin/convince-your-boss-to-go-serverless-at-aws-user-group-tirupathi-and-serverless-bangalore-meetup)
+   - Video Stream: [https://www.youtube.com/watch?v=yzut-TDCqmk](https://www.youtube.com/watch?v=yzut-TDCqmk)
+   - Resource Person: [Vadym Kazulkin](https://twitter.com/VKazulkin), Head of Technology Strategy at ip.labs GmbH
+
+
+#### Session 2: CI CD in the World of Serverless
```

Comment:
session 2 should come above session 1. We will follow chronological order

Reply:
done